index.html

```diff
@@ -298,41 +301,38 @@
 
 
   d3.select("#sort-asc")
-      .on("click", function(){
-        d3.selectAll('#sorting-buttons button').classed('active', false);
-         
-          if (options.sorted === 'asc') {
-            options.sorted === false;
-          } else {
-            options.sorted ==='asc';
-
-          d3.select('#sort-asc').classed('active',true);
-          }
+      .on("click", function(d){
+           	 
+		    if (options.sorted === 'asc') {
```

Comment:
The issue is that your function is `return`ing. Once Javascript hits a `return` statement, it'll return that value (in this case to the browser, which is paying it no mind) and stop reading the function.